src/parsers.js

```diff
@@ -26,7 +27,11 @@ export function parseVector3(str) {
  * Returns a parsed vector2 object.
  */
 export function parseVector2(str) {
-  return str.isVector2 ? str : new Vector2(...str.trim().split(/\s+/).map((elm) => parseFloat(elm)));
+  if (str.isVector2) return str;
+  if (Array.isArray(str) && str.length === 2) {
```

Comment:
@WTDuck Thank you for your many PRs.
I think it does not need `str.length === 2` since THREE.Vector2 treats undefined as 0, and ignores after 2nd arguments.
When `str` is an array and its length is not 2, this function will call `trim()` on the array and causes an error.